### PR TITLE
DerpQuest: Update Italian translation

### PR DESCRIPTION
**Pull request by Pietro395**

### CODE REVIEW

File: res/values-it/derpquest_strings.xml [hunk 1]
```diff
@@ -30,14 +33,23 @@
     <string name="theming_title">Temi</string>
     <string name="animations_title">Animazioni</string>
     <string name="nav_bar_category_title">Barra di navigazione</string>
+    <string name="statusbar_notifi_cat">Notifiche</string>
```
Comment (idoybh) [inline res/values-it/derpquest_strings.xml:36]: this string is defined twice. make sure you've got the right one

**Review by idoybh — CHANGES_REQUESTED** [review]

File: res/values-it/derpquest_strings.xml (hunk 1 above)
Comment (idoybh) [reply res/values-it/derpquest_strings.xml:36]: prev definition in line 796

Comment (Pietro395) [reply res/values-it/derpquest_strings.xml:36]: The one in line 796 is wrong, can you delete it? Thank you

Comment (idoybh) [reply res/values-it/derpquest_strings.xml:36]: Yes will delete and merge now :) Thanks again

**Review by idoybh — APPROVED** [review]

**Pull request merged**